src/messaging/tests/TestExchangeMgr.cpp

```diff
@@ -106,14 +107,14 @@ void CheckNewContextTest(nlTestSuite * inSuite, void * inContext)
     err = exchangeMgr.Init(&conn);
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
 
-    ExchangeContext * ec1 = exchangeMgr.NewContext(kSourceNodeId, (void *) 0x1234);
+    ExchangeContext * ec1 = exchangeMgr.NewContext(kSourceNodeId, (ExchangeContextDelegate *) 0x1234);
```

Comment:
maybe create it as a valid mock delegate on the stack. Raw casts to invalid objects seem wrong.